src/screens/TopicPageScreen/index.js

```diff
@@ -474,6 +474,7 @@ const TopicPageScreen = (props) => {
         screen="topic_screen"
       />
       <BottomSheetFollow
+        topicId={topicId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `topicId` as a prop to `BottomSheetFollow` is a good change, assuming that the `BottomSheetFollow` component has been updated to accept and use this prop. This allows for more specific behavior in the `BottomSheetFollow` component based on the topic ID.
```diff
       <BottomSheetFollow
+        topicId={topicId}
         ref={bottomSheetFollowRef}
         onClose={() => bottomSheetFollowRef.current.close()}
         topicName={topicDetail.name}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->